docs/guides/faq.md

```diff
@@ -0,0 +1,91 @@
+## FAQ
+
+**Q: What is video.js?**
```

Comment:
I think each question should be a heading.
So, at the top, FAQ should be `#` and each question should be `##`.
```markdown
# FAQ

## What is video.js?
Video.js is a video player.
```